### PR TITLE
[WIP][8.0][BR000973][T22092]Refactor elico_project into several distinct modules[project_issue_id_display]

### PR DESCRIPTION
**Pull request by dongshukai**
This PR is for project_issue_id_display

### CODE REVIEW

File: project_issue_id_display/README.rst [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+To install this module, you need to:
+
+ * have basic modules installed (project_issue)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
```
Comment (elicoidal) [inline project_issue_id_display/README.rst]: check https://github.com/Elico-Corp/odoo-addons/pull/171/files

File: project_issue_id_display/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (http://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Project Issue ID Display',
+    'version': '8.0.1.0.1',
```
Comment (elicoidal) [inline project_issue_id_display/__openerp__.py]: 8.0.1.0.0

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details

**Comment by dongshukai** [comment]
@Elico-Corp/odoo-reviewers 
pls check

**Review by elicoidal — APPROVED** [review]

**Pull request merged**